decentraland/components/presentational/MyAccount.test.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { shallow } from "enzyme";
+import MyAccount from "./MyAccount";
+import { accountCreationActions } from "../../helpers/testHelpers";
+
+describe("MyAccount", () => {
+  it("renders the component", async () => {
+    expect(
+      shallow(
+        <MyAccount progress={0.5} creationActions={accountCreationActions} />
```

Comment:
Hey @mpetrunic, as code style we are declaring props before using them whenever is possible.